qemu/tests/cfg/blockdev_snapshot_install.cfg

```diff
@@ -0,0 +1,31 @@
+- blockdev_snapshot_install:
+    type = blockdev_snapshot_install
+    virt_test_type = qemu
+    need_install = yes
+    start_vm = no
+    image_name_image1 = "images/snapshot_install"
```

Comment:
remove this line for 
params.update(
        {"image_name_%s" % base_image: params["image_name"],
Then image_name_image1 never takes effect.

Reply:
have updated.